tools/cibadmin.c

```diff
@@ -698,11 +698,10 @@ main(int argc, char **argv)
     crm_trace("%s exiting normally", crm_system_name);
 
     free_xml(input);
-    rc = the_cib->cmds->signoff(the_cib);
+    rc = cib__clean_up_connection(&the_cib);
```

Comment:
Since this now returns a standard code, we need to change crm_errno2exit() to pcmk_rc2exitc() below

Reply:
Ah right, fixed.